src/app/core/data/request.models.ts

```diff
@@ -230,7 +231,87 @@ export class IntegrationRequest extends GetRequest {
     return IntegrationResponseParsingService;
   }
 }
+
+export class SubmissionFindAllRequest extends GetRequest {
```

Comment:
Please add TypeDoc/comments to describe *all the new classes in this file* (there are several more in this same file that have no comments to describe them.